test/cloud_platform.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+# Enum for cloud platforms supported by Snowflake (AWS/AZURE/GCP)
+class CloudPlatform(Enum):
+    AWS = 1
+    AZURE = 2
+    GCP = 3
+    ALL = 4
```

Comment:
I am not sure if that is useful in this particular case but maybe it should be flags?

Reply:
Thanks, @sfc-gh-akowalczyk already mentioned that. I don't plan to mix values eg. run test on AWS and AZURE so I decided to leave it as an ordinary enum.